app/assets/stylesheets/modules/sitter.scss

```diff
@@ -129,13 +151,25 @@
         font-weight: $sitter-headline-weight;
       }
 
+      #image-upload {
+        ul li .postfix {
+          width: 30%;
+        }
+      }
+
       .image-div {
         width: 15em;
+        height: 15em;
+
+        li {
+          background: #000;
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.